### PR TITLE
Test of changes to MDL CDL file

### PR DESCRIPTION
**Pull request by jjlevit**
Formatting changes to MDL file

### CODE REVIEW

File: lib/bald/tests/integration/CDL/ProcessChain0300.cdl [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        :process_chain = "gfsmos_process_chain";
+        :bald__isPrefixedBy = "prefix_list" ;
+        :bald__isAliasedBy = "alias_list" ;
+
+group: alias_list {
+
+// group attributes:
+        :DA__ = "https://codes.nws.noaa.gov/DataAssimilation" ;
+        :NWP__ = "https://codes.nws.noaa.gov/NumericalWeatherPrediction" ;
+        :StatPP__ = "https://codes.nws.noaa.gov/StatisticalPostProcessing" ;
```
Comment (marqh) [inline lib/bald/tests/integration/CDL/ProcessChain0300.cdl]: you will also require
`       :bald__ = "http://binary-array-ld.net/latest/" ;`

**Comment by jjlevit** [comment]
Tests passed.

**Comment by jjlevit** [comment]
This code and sample CDL file is ready to merge with the master branch.

**Comment by marqh** [comment]
Hi @jjlevit 

this looks grand and ready to merge, thank you

mark

**Pull request merged**